test/train-sets/ref/cats-pdf-train.stderr

```diff
@@ -13,14 +13,14 @@ power_t = 0.5
 Enabled reductions: ftrl-Coin Betting, scorer-identity, binary, cats_tree, get_pmf, pmf_to_pdf, cb_explore_pdf, cats_pdf
 Input label = continuous
 Output pred = pdf
-average  since         example        example  current  current  current
-loss     last          counter         weight    label  predict features
-0.657567 0.657567            1            1.0 {185.121,0.657567,6.20426e-05}                10
```

Comment:
Looks like not just reformatting, but some information loss here. Is it intentional?

Reply:
Yes, this is intentional. If the label or prediction overflow their column they are cut off. In this particular example the predictions were empty but that was hidden by the overflow of the old example. 

We can instead allow label/prediction to overflow to the next line(s) so no information is lost. What do you think?

Reply:
not sure actually. Can we maybe just expand columns limit a bit? with current approach seems like output for cats is always useless.
Allowing overflowing to the next line may complicate parsing of this output (although it is not a great idea anyway)

Reply:
In this PR I expanded label/pred columns from 8 to 14 chars. I can do more, but they are still not going to fit I think. The other issue here is that when we serialize these predictions and labels we don't reduce the decimal precision so they often have 7 or so digits after the decimal place and take up a lot of width. There is also no notion of the canonical string representation of a prediction which I feel would be handy.

Reply:
yeah that would be great. seems to be something that could be done as followup work though